service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -413,6 +413,7 @@ public ApplicationsBundle getApplicationBundleByUUID(UUID elementId) {
     private final Selector childSelector;
     private final Selector careOrderSelector;
     private final Selector newHearingSelector;
+    private final Selector appointedGuardianSelector;
```

Comment:
is appointedGuardians_label anywhere in the model? Is not needed?

Reply:
AppointedGuardianBlockPrePopulator was missing tests, will add label to model so can test it more easily

Reply:
actually don't need it in casedata